docs/emj/docs/features.md

```diff
@@ -0,0 +1,203 @@
+# 📇 🏠
+
+## 📇 🏠 FastAPI
+
+**FastAPI** 💁‍♂️ 👆 📄:
+
+### ⚙️ 🈺 🐩
+
+* <a href="https://github.com/OAI/OpenAPI-Specification" class="external-link" target="_blank"><strong>OpenAPI</strong></a> 🏗 API, ✅ 🙂📛 <abbr title="💭   HTTP-👩‍🔬 ✅ : POST, GET, PUT, DELETE">🛠</abbr> <abbr title="💭: 🔗, 🛣, '🍵', ♒️.">🌌</abbr>, 🔢, ✔ 💪, 💂‍♂, ♒️.
+
```

Comment:
I think "Security" in native emoji is more like "👮" then your translated "💂‍♂". However, the other parts is perfect, I like it